bedrock/firefox/templates/firefox/includes/structured-data-firefox-faq.html

```diff
@@ -57,14 +57,6 @@
                     "text": "Not only is Firefox safe to use, it also helps keep your data and private information safe. The Firefox browser automatically blocks known third party trackers, social media trackers, cryptominers and fingerprinters from collecting your data. Learn more about the privacy in our products."
                 }
             },
-            {
```

Comment:
Instead of removing the canary, why not replace it with the new answer?
```
            {
                "@type": "Question",
                "name": "Does Firefox sell your personal data?",
                "acceptedAnswer": {
                    "@type": "Answer",
                    "text": "Yes. That’s a promise. "
                }
            },
```